app/src/main/java/org/oppia/app/player/state/StateViewModel.kt

```diff
@@ -53,7 +57,12 @@ class StateViewModel @Inject constructor() : ObservableViewModel() {
   private fun getPendingAnswerWithoutError(
     statePlayerRecyclerViewAssembler: StatePlayerRecyclerViewAssembler
   ): UserAnswer? {
-    val answerHandler = statePlayerRecyclerViewAssembler.getPendingAnswerHandler(itemList)
+    val items = if (isSplitView.get() == true) {
```

Comment:
Is ``if (isSplitView.get()) {`` not sufficient?

Reply:
It is complaining when I do it this way because `isSplitView.get()` is nullable (Boolean?)  
So I have done it this way. I'm not sure whether using `isSplitView.get()!!` will be suitable here or it may cause the app to crash.

Reply:
Resolving this on behalf of Ben.